image_proc/launch/image_proc.launch.py

```diff
@@ -30,36 +30,68 @@
 # ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 # POSSIBILITY OF SUCH DAMAGE.
 
-from launch import LaunchDescription
-from launch_ros.actions import ComposableNodeContainer
+import sys
+
+from launch import LaunchDescription, LaunchIntrospector, LaunchService
+from launch_ros import actions, get_default_launch_description
+
 from launch_ros.descriptions import ComposableNode
 
 
 def generate_launch_description():
-    return LaunchDescription([
-        ComposableNodeContainer(
-            package='rclcpp_components', node_executable='component_container',
-            node_name='image_proc_container', node_namespace='',
-            composable_node_descriptions=[
-                ComposableNode(
-                    package='image_proc',
-                    node_plugin='image_proc::DebayerNode',
-                    node_name='debayer'
-                ),
-                ComposableNode(
-                    package='image_proc',
-                    node_plugin='image_proc::RectifyNode',
-                    node_name='rectify_mono'
-                ),
-                ComposableNode(
-                    package='image_proc',
-                    node_plugin='image_proc::RectifyNode',
-                    node_name='rectify_color',
-                    remappings=[
-                        ('image_mono', 'image_color'),
-                        ('image_rect', 'image_rect_color')
-                    ]
-                ),
-            ],
-        ),
-    ])
+    ld = LaunchDescription()
+
+    # Load composable container
+    image_processing = actions.ComposableNodeContainer(
+        node_name="image_proc_container",
+        package='rclcpp_components',
+        node_executable='component_container',
+        composable_node_descriptions=[
+            ComposableNode(
+                package='image_proc',
+                node_plugin='image_proc::DebayerNode',
+                node_name='debayer_node',
+            ),
+            # Example of rectifying an image
+            ComposableNode(
+                package='image_proc',
+                node_plugin='image_proc::RectifyNode',
+                node_name='rectify_mono_node',
```

Comment:
`rectify_mono_node` but then you have the topic remap to `image_rect_color`. Looking at the other launch file, it looks like it has 2 rectify nodes: one for mono and the other for color.